packages/nala/build.sh

```diff
@@ -32,42 +26,20 @@ termux_step_pre_configure() {
 		${_CROSSENV_PREFIX}
 	popd
 	. ${_CROSSENV_PREFIX}/bin/activate
-
-	build-pip install tomli
 }
 
 termux_step_make_install() {
-	python setup.py install --force
-
 	export PYTHONPATH=$TERMUX_PREFIX/lib/python${_PYTHON_VERSION}/site-packages
-	python setup.py install --force --prefix $TERMUX_PREFIX
-
-	pushd $PYTHONPATH
-	_NALA_EGG=
-	for f in nala-${TERMUX_PKG_VERSION##*:}-py${_PYTHON_VERSION}.egg; do
-		if [ -e "$f" ]; then
-			_NALA_EGG="$f"
-			break
-		fi
-	done
-	test -n "${_NALA_EGG}"
-	popd
+	python -m pip install . --force --prefix $TERMUX_PREFIX
 
 	install -Dm600 -t $TERMUX_PREFIX/etc/nala debian/nala.conf
 }
 
 termux_step_create_debscripts() {
 	cat <<- EOF > ./postinst
 	#!$TERMUX_PREFIX/bin/sh
-	echo "./${_NALA_EGG}" >> $TERMUX_PREFIX/lib/python${_PYTHON_VERSION}/site-packages/easy-install.pth
 	mkdir -p $TERMUX_PREFIX/var/lib/nala
 	mkdir -p $TERMUX_PREFIX/var/log/nala
-	echo "Installing dependencies through pip..."
-	pip3 install anyio httpx jsbeautifier pexpect rich
```

Comment:
Why is this removed? If there are python dependencies then they need to be installed through pip

Reply:
Was python-debian a runtime dependency? If yes then it should be installed here as well

Reply:
If you install package with `pyproject.toml` with pip, all the dependencies will be installed alongside it.
I tested both in the termux build docker and installing using the patches on termux.
(Installing the runtime dependencies explicitly is still needed in packages that use legacy build formats like `setup.py` and `setup.cfg`)
![image](https://user-images.githubusercontent.com/65723952/193433333-37c161d0-4b90-403e-9dcd-1858c21db93d.png)
![image](https://user-images.githubusercontent.com/65723952/193433588-d061e1f9-eb85-4679-aec7-e3664e23381d.png)

Reply:
`python-debian` is a runtime dependency that @volitank forgot to add to the `pyproject.toml` file (since it's installed by default on debian-based distros), but will probably be on it in the next release.